sdk/sdk-api/src/main/java/mil/emp3/api/KML.java

```diff
@@ -39,15 +38,15 @@
     /**
      * This constructor creates a KML feature from a document geo object.
      * @param document An object that implements the IGeoDocument interface.
-     * @throws InvalidParameterException This exception is raised if the geo renderable object does not contain a KML string or URL.
+     * @throws IllegalArgumentException This exception is raised if the geo renderable object does not contain a KML string or URL.
      * @throws XmlPullParserException This exception is raised if the KML fails to parse correctly.
      * @throws IOException This exception is raised in the event of an IO error accessing the URL.
      */
-    public KML(IGeoDocument document) throws InvalidParameterException, XmlPullParserException, IOException {
+    public KML(IGeoDocument document) throws IllegalArgumentException, XmlPullParserException, IOException {
```

Comment:
As stated in the Java Language Specification, "It is permitted but not required to mention unchecked exception classes in a throws clause".